docs/user/interop.rst

```diff
@@ -145,7 +145,7 @@ One can wrap a function in a nicer API like:
    import scala.scalanative.unsafe._
 
    def myprintf(format: CString, args: CVarArg*): CInt =
-     Zone { implicit z =>
+     Zone { 
```

Comment:
The description in lines 311 thru 315 is excellent. For me,
it is an immediate  "See one, do one".

I understand that, for a number of reasons, we do not want
to duplicate text. 

Most people read docs, at from earlier to later, even when 
searching for reminders "I know the concept exists, were is
is it described in detail?".   

Similar to expanding an acronym on first use, this will be,
I think the first exposure to `Zone`.

Perhaps adding a short comment here referring to 
the "Memory Management" example.

`Zone { // Scala 3, see Scala 2 Zone.alloc() in Memory Management section below.